src/main/drivers/bus_spi.c

```diff
@@ -240,6 +240,25 @@ void spiBusSetInstance(busDevice_t *bus, SPI_TypeDef *instance)
     bus->busdev_u.spi.instance = instance;
 }
 
+uint16_t spiCalculateDivider(uint32_t freq)
+{
+#if defined(STM32F4) || defined(STM32G4) || defined(STM32F7)
+    uint32_t spiClk = SystemCoreClock/2;
+#elif defined(STM32H7)
+    uint32_t spiClk = 100000000;
+#endif
```

Comment:
Probably best to add an error catch-all to throw when a new architecture is added.

Reply:
Done